config/clusters/2i2c-aws-us/go-bgc.values.yaml

```diff
@@ -33,7 +33,6 @@ jupyterhub:
       GitHubOAuthenticator:
         oauth_callback_url: https://go-bgc.2i2c.cloud/hub/oauth_callback
         allowed_organizations:
-          - 2i2c-org:hub-access-for-2i2c-staff
```

Comment:
Removing ourselves from `allowed_organization` is fine, because we are added to `admin_users`.

Doing this also isn't influencing the filtering of profile_list entries using our basehub injected `allowed_teams` config.